lib/internal/util/trace_timer.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+const {
+  MathFloor,
+  NumberPrototypeToFixed,
+  Number,
+  StringPrototypeSplit,
+  StringPrototypePadStart,
+} = primordials;
+
+const { trace } = internalBinding('trace_events');
```

Comment:
Note that adding an unsnapshotted new file leads to a performance hit on first load. It doesn't seem necessary to split a new file for these, they can probably be just moved into `lib/internal/util/debuglog.js`

Reply:
I split in another file to be able to reuse this functionality on `Console`, and `Console` don't need to import `debuglog`.

Reply:
In the end, I moved to the `debuglog` since I introduced `flags` to the `timer` functions to skip calling C++.